### PR TITLE
Version 1 of validator_info design

### PR DESCRIPTION
**Pull request by dsurnin**
Signed-off-by: dsurnin <dmitry.surnin@dsr-corporation.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

File: design/validator_info.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        "indy-node": "1.3.319",
+        "sovrin": null
+    },
+    
+    "alias": "Node1" # node's name
+}
+```
+
+## Modification - New Read Command
+Validator_info should be accessible as read command, available for TRUSTEE only. New command should provide info from
```
Comment (ashcherbakov) [inline design/validator_info.md]: I think it should be accessible by STEWARDS as well

File: design/validator_info.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+all the connected nodes without need of consensus (similar to force=True flag in upgrade cmd). Command should allow
+requesting os all parameters or some subset of parameters.
```
Comment (ashcherbakov) [inline design/validator_info.md]: os - typo

File: design/validator_info.md (hunk 1 above)
Comment (dsurnin) [reply design/validator_info.md]: changed

File: design/validator_info.md (hunk 2 above)
Comment (dsurnin) [reply design/validator_info.md]: fixed

**Comment by ashcherbakov** [comment]
test this please

**Comment by ashcherbakov** [comment]
test this please

File: design/validator_info.md (hunk 1 above)
Comment (mgbailey) [inline design/validator_info.md]: Please add the following from INDY-967:
- the BLS key
- the timezone

**Pull request merged**